descriptors/ModuleDescriptor-template.json

```diff
@@ -795,6 +795,24 @@
           "delay": "1"
         }
       ]
+    },
+    {
+      "id": "subscribe-callback",
```

Comment:
Is this an arbitrary name for grouping all of the subscription endpoints together?

Reply:
yes

Reply:
Given this is defined as an interface, unless it is marked multiple (which would confuse mod-pubsub I imagine), it can only be implemented by a single module at a time.

I think we should consider giving this a circulation specific name in order to avoid accidental conflicts between other modules. 

(I wonder how this was addressed in mod-inventory?)

Reply:
Do you mean something like `"circulation-subscribe-callback"`?

> (I wonder how this was addressed in mod-inventory?)

It is defined as `inventory-event-handlers`

Reply:
Yes, however, as FoliJet used `event-handlers` in inventory, I think it might be worth being consistent until we decide on a convention

Reply:
Renamed to `circulation-event-handlers` v0.1